src/frontend/Player/createVideojsPlayer.ts

```diff
@@ -95,10 +95,12 @@ export const createVideojsPlayer = (
     sources,
   };
 
-  const player = videojs(videoNode, options, () => {
-    if (onReady) {
-      onReady(player);
+  const player = videojs(videoNode, options, function () {
+    if (isLive) {
+      this.play();
```

Comment:
I'm not convinced this is working.

Reply:
Sources are defined, according to the API's documentation it should (in theory) work. I think we will have no choices but to test ourself

Reply:
![image](https://user-images.githubusercontent.com/83247226/182407401-0c5fc9d6-b1a0-4fb1-ba8d-218fb88566f9.png)

Sources are defined, so according to the API's documentation it should (in theory) work. I think we will have no choices but to test ourself

Reply:
After few tests using browserstack, it doesn't work

Reply:
When I tested it on safari it does not works because on iOS with safari you need a userInput to start the player. 
Yet, according to this [blog](https://blog.videojs.com/autoplay-best-practices-with-video-js/) we are increasing the chance for it to works by calling the play method. At best, it causes the player to start, at worse nothing changes and the player does not start.

Reply:
We can test with that indeed.